### PR TITLE
Added user model + db migration to power email subscriptions

### PR DESCRIPTION
**Pull request by danthaman44**
```
danthaman-mac:ruby danthaman$ heroku local:run bundle exec rspec spec/models/user_spec.rb
[OKAY] Loaded ENV .env File as KEY=VALUE Format
...

Finished in 0.17486 seconds (files took 1.67 seconds to load)
3 examples, 0 failures
```

### CODE REVIEW

File: db/migrate/20200510012452_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :email, unique: true
```
Comment (danthaman44) [inline db/migrate/20200510012452_create_users.rb:4]: is `unique: true` necessary if the model already enforces uniqueness for the field?

Comment (jtmckibb) [reply db/migrate/20200510012452_create_users.rb:4]: It's good to have both DB and model enforcement

Comment (jtmckibb) [reply db/migrate/20200510012452_create_users.rb:4]: But crazy, can we make this optional? I think we designed the same model: See "Contact"

https://docs.google.com/document/d/1pbP1NX2bdH_aZGbBJxwxWUfvIf8npKSn0yrDftBGvz8/edit#

**Comment by Yong-L** [comment]
Nope doesn’t matter for me! LGTM

File: db/migrate/20200510012452_create_users.rb (hunk 1 above)
Comment (danthaman44) [reply db/migrate/20200510012452_create_users.rb]: Done

**Pull request merged**